### PR TITLE
Let children onFocus onBlur events in TouchableWithoutFeedback pass through to parent

### PR DESCRIPTION
**Pull request by kaiguo**
Fixes #3927. 

TextInput's onFocus and onBlur callbacks are not functioning, this is because we [wrapped RCTTextInput in a TouchableWithoutFeedback](https://github.com/microsoft/react-native-windows/blob/master/vnext/src/Libraries/Components/TextInput/TextInput.windows.js#L1013-L1025), and onFocus onBlur were [overridden by TouchableWithoutFeedback](https://github.com/microsoft/react-native-windows/blob/master/vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js#L345-L346).

To fix this we should let [OVERRIDE_PROPS](https://github.com/kaiguo/react-native-windows/blob/38bc6cad5c3cdd7d1e00ac0e3a7185e12a2fa381/vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js#L313-L316) do the work and just copy the rest of the children props to parent.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4078)

### CODE REVIEW

File: packages/E2ETest/wdio/test/testInput.spec.ts [hunk 1]
```diff
@@ -13,6 +13,16 @@ beforeAll(() => {
...
+  it('Click on TextInput to focus', () => {
+    TextInputTestPage.clickTextInput();
+    assert.ok(TextInputTestPage.getTextInputCurText().includes('onFocus'));
+  });
+
+  it('Click on multiline TextInput to move focus away', () => {
+    TextInputTestPage.clickMultilineTextInput();
+    assert.ok(TextInputTestPage.getTextInputPrevText().includes('onBlur'));
+  });
+
```
Comment (ddalp) [inline packages/E2ETest/wdio/test/testInput.spec.ts]: Thanks for adding test coverage!

File: vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js [hunk 2]
```diff
@@ -342,8 +342,6 @@ const TouchableWithoutFeedback = ((createReactClass({
       onResponderTerminate: this.touchableHandleResponderTerminate,
       onKeyUp: this._onKeyUp,
       onKeyDown: this._onKeyDown,
```
Comment (ddalp) [inline vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js]: There are more events removed in RN61 TouchableWithoutFeedBack.js, for example: onKeyUp and onKeyDown, should we remove those events too?

Comment (kaiguo) [reply vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js]: I think we should, but looks like ReactNative is not doing it? https://github.com/facebook/react-native/blob/master/Libraries/Components/Touchable/TouchableWithoutFeedback.js#L75-L94

Removing it here and adding it to the override list basically means if there's no event handler set on Touchable, use the callbacks on children to handle the events, which I think makes sense for keyUp/Down events, but I'm not sure why they are not in ReactNative.

Comment (kaiguo) [reply vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js]: Oh, is keyUp/Down windows only?

Comment (ddalp) [reply vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js]: I think so. Search for topKeyDown definition.

---
In reply to: [377886796](https://github.com/microsoft/react-native-windows/pull/4078#discussion_r377886796) [](ancestors = 377886796)

Comment (kmelmon) [reply vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js]: yes keyUp/keyDown are windows-specific.  See [https://github.com/microsoft/react-native-windows/pull/2623](https://github.com/microsoft/react-native-windows/pull/2623).

---
In reply to: [377889221](https://github.com/microsoft/react-native-windows/pull/4078#discussion_r377889221) [](ancestors = 377889221,377886796)

Comment (kaiguo) [reply vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js]: Looks like we are re-routing the events to onPress so we probably shouldn't remove them.

https://github.com/microsoft/react-native-windows/blob/master/vnext/src/Libraries/Components/Touchable/TouchableHighlight.windows.js#L453-L474

Comment (kaiguo) [reply vnext/src/Libraries/Components/Touchable/TouchableWithoutFeedback.windows.js]: @ddalp just checked there's no keyDown/Up events on TextInput, there's only onKeyPress. We are sending out the onPressed event in _onKeyUp callback, so I think we should keep it as is.

Also added a test to verify keyPress event on TextInput.

**Comment by jagorrin** [comment]
When can I expect this fix to be included in a release?

**Comment by kmelmon** [comment]
@jagorrin The fix is available now, but the published release it's part of is in beta (0.61-beta).  Version 61 will likely stay in beta for at least another week.  Do you need this to be part of a previous release?

**Comment by jagorrin** [comment]
Would it be possible for this to be cherry-picked into a new 0.60-vnext release? It would be great to be able to have this fix without having to update to 61 right now to get it.

If so, I have two other fixes that are in the same spot - available only in the 0.61-beta releases, but would be great to have in a new 0.60-vnext release:
- https://github.com/microsoft/react-native-windows/pull/4061
- https://github.com/microsoft/react-native-windows/pull/4043

**Comment by kmelmon** [comment]
@jagorrin all that's required is to submit these as PRs into the 0.60-stable branch.  Once checked in, our CI loop will run and publish to npm.  If you can prepare the PRs, I'd be happy to review and approve.

**Pull request merged**